packages/docs/pages/build-integrations/auth.md

```diff
@@ -0,0 +1,195 @@
+# Auth
+
+:::info
+
+The build integrations section is best understood when read from beginning to end. To get the most value out of it, start from the first page and read through page by page.
+
+1. [Folder structure](/build-integrations/folder-structure)
+2. [App](/build-integrations/app)
+3. [Global variable](/build-integrations/global-variable)
+4. [<mark>Auth</mark>](/build-integrations/auth)
+5. [Triggers](/build-integrations/triggers)
+6. [Actions](/build-integrations/actions)
+7. [Examples](/build-integrations/examples)
+
+:::
+
+## Sign up for the cat API
+
+Go to the [sign up page](https://thecatapi.com/signup) of the cat API and register your account. It allows you to have 10k requests per month with a free account. You will get an API key by email after the registration. We will use this API key for authentication later on.
+
+## The cat API docs
+
+You can find detailed documentation of the cat API [here](https://docs.thecatapi.com). You need to revisit this page while building the integration.
+
+## Add auth to the app
+
+Open the `thecatapi/index.ts` file and add the highlighted lines for authentication.
+
+```typescript{2,13}
+import defineApp from '../../helpers/define-app';
+import auth from './auth';
+
+export default defineApp({
+  name: 'The cat API',
+  key: 'thecatapi',
+  iconUrl: '{BASE_URL}/apps/thecatapi/assets/favicon.svg',
+  authDocUrl: 'https://automatisch.io/docs/apps/thecatapi/connection',
+  supportsConnections: true,
+  baseUrl: 'https://thecatapi.com',
+  apiBaseUrl: 'https://api.thecatapi.com',
+  primaryColor: '000000',
+  auth,
```

Comment:
Do we mention the `beforeRequest` middleware to incorporate the authentication header in the HTTP client?